docs/_docs/developers-guide/clients/jdbc-driver.adoc

```diff
@@ -72,6 +72,11 @@ jdbc:ignite:thin://host[:port][,host[:port][/schema][[?parameter1=value1][;param
 ** `partitionAwarenessMetadataCacheSize` - Size of cache to store partition awareness metadata of queries, in number of entries. Default value: `1024`.
 ** `queryTimeout` - Number of seconds the driver will wait for a `Statement` object to execute. 0 means there is no limit. Default value: `0`.
 ** `connectionTimeout` - Number of milliseconds JDBC client will wait for server to respond. 0 means there is no limit. Default value: `0`.
+** `backgroundReconnectInterval` - Background reconnect interval, in milliseconds. The value `0` can be used to disable background reconnection. Default value is `30000` (30 seconds).
+Background reconnection is used to restore old and establish new connections to cluster nodes in the background.
+For example, if you disable it, the client will not try to restore connections to temporarily unavailable cluster nodes.
+** `reconnectRetriesLimit` - Maximum number of *retry* attempts to establish connection. The value `0` means that no retries will be made. The value `-1` means that the number of retries is not limited. Default value is `16`.
```

Comment:
Let`s suppose we have long living (as in practice))) ) cluster with temporary connection failures, as i can see now it counts for 16 and will stop to reconnect ? As for me it looks like very strange - client need to try to connect until success. Probably there is some logic in such a design but currently it look like strange for me.

Reply:
I decided to remove this property, since it should not have effect for JDBC

Reply:
But as for your question, no, the retry counter applies only to a specific operation.
Let's say the connection to the server was interrupted while retrieving the list of tables, and if the retry counter is greater than 0, we will still try to retry the operation. Next time, if the connection is broken again, the counter starts from zero again.